### DISCUSSION TITLE
who is opusforlife2?

### DISCUSSION THREAD

**Discussion by fenopa** [TeamNewPipe/NewPipe#10672] [Questions] [answered]
Why can he act like a child and abruptly close questions of people? Who gives him this right?

**Comment by fenopa** [comment]
@theScrabi @Stypox @TobiGr please take a look at this abusive behaviour and take back these rights from this person.

**Comment by Stypox** [comment, marked as answer]
opusforlife2 is a team member and he usually takes care of issues by classifying them, closing duplicates, and making sure users provide enough information. He does a valuable job for us developers, who would not have enough time to look into the many low-quality issues opened.
